modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -72,6 +78,13 @@ pullSecret: '{"auths": ...}'
 sshKey: ssh-ed12345 AAAA...
 ----
 
+<1> Setting this option to `false` enables preallocation of disks. The default is `true`. Setting `sparse` to `true` with `format` set to `raw` is not available for block storage domains. The 'raw' format writes the entire virtual disk to the underlying physical disk.
+<2> Can be set to `cow` or `raw`. The default is `cow`. The 'cow' format is optimized for virtual machines.
```

Comment:
The same applies to one instance of the code item "cow" in this line. Please update it to use back-ticks instead of single quotes.